### PR TITLE
Add Clarification for Enabling SSH on MacOS

### PR DESCRIPTION
**Pull request by bennettforkner**

### CODE REVIEW

File: docs/Developer's Guide.md [hunk 1]
```diff
@@ -97,6 +97,8 @@ Sometimes, you would like to connect the frontend on your local computer to the
    [Ubuntu Installation](ubuntu.com/server/docs/service-openssh)
 
    MacOS & other Linux distributions should already have an SSH server installed.
+   
+   On MacOS, you may have to enable the SSH server in System Preferences. Follow [these instructions](https://osxdaily.com/2022/07/08/turn-on-ssh-mac/) to do so.
```
Comment (EjPlatzer) [inline docs/Developer's Guide.md:101]: I appreciate you clarifying these instructions, but the linked article gives incorrect instructions.

As of MacOS Ventura, the `Remote Login` option is under `General` > `Sharing` > `Remote Login`, whereas the article says to go to `General` > `General`> `Remote Login`, which is confusing.

I also don't like the idea of relying on third-party websites for documentation unless it is the third-party that controls the thing they're documenting.

Apple has documentation on enabling Remote Login here: https://support.apple.com/guide/mac-help/allow-a-remote-computer-to-access-your-mac-mchlp1066/mac

**Review by EjPlatzer — CHANGES_REQUESTED** [review]

File: docs/Developer's Guide.md (hunk 1 above)
Comment (bennettforkner) [reply docs/Developer's Guide.md:101]: Ah thanks. I couldn't test. 
```suggestion
   On MacOS, you may have to enable the SSH server in System Preferences. Follow [these instructions](https://support.apple.com/lt-lt/guide/mac-help/mchlp1066/mac) to do so.
```

Comment (EjPlatzer) [reply docs/Developer's Guide.md]: Hmm, somehow you linked to the Lithuanian version of the docs (note the `lt-lt` in the URL). The direct (locale-agnostic) link is: https://support.apple.com/guide/mac-help/allow-a-remote-computer-to-access-your-mac-mchlp1066/mac

File: docs/Developer's Guide.md [hunk 2]
```diff
@@ -97,6 +97,8 @@ Sometimes, you would like to connect the frontend on your local computer to the
    [Ubuntu Installation](ubuntu.com/server/docs/service-openssh)
 
    MacOS & other Linux distributions should already have an SSH server installed.
+   
+   On MacOS, you may have to enable the SSH server in System Preferences. Follow [these instructions](https://support.apple.com/lt-lt/guide/mac-help/mchlp1066/mac) to do so.
```
Comment (bennettforkner) [inline docs/Developer's Guide.md:101]:
```suggestion
   On MacOS, you may have to enable the SSH server in System Preferences. Follow [these instructions](https://support.apple.com/guide/mac-help/allow-a-remote-computer-to-access-your-mac-mchlp1066/mac) to do so.
```

**Review by EjPlatzer — APPROVED** [review]
LGTM!

**Comment by jakedcolbert** [comment]
Looks good to me. The apple reference is the most helpful for locating the remote login setting. Thank you, Evan and Bennett!

**Review by jakedcolbert — APPROVED** [review]

**Pull request merged**